metadata.json

```diff
@@ -61,6 +61,12 @@
       ]
     },
     {
+      "operatingsystem": "OpenSuSE",
```

Comment:
Sorry, this will not be a supported platform. The changes to allow ntp to work on OpenSuSE are fine. But OpenSuSE isnt a supported platform.